lib/board.rb

```diff
@@ -80,4 +80,13 @@ def vertical_submarine
     new_coords
   end
 
+  def place(ship, coordinates)
+    if valid_length?(ship, coordinates) && consecutive?(ship, coordinates)
+
+      coordinates.each do |cell_key|
+        @cells[cell_key].place_ship(ship)  
```

Comment:
I like how you're utilizing the coordinates in the test to draw out the coordinates in our cells array and access methods in the cells class without having to instantiate a new object!